### PR TITLE
That mk widget...

### PR DESCRIPTION
**Pull request by daonb**
It's pretty much done. There's one js file that the user need to add to his template to add a pop-up to all mk-names.

### CODE REVIEW

File: knesset/settings.py [hunk 1]
```diff
@@ -306,3 +306,4 @@
     from local_settings import *
 except ImportError:
     pass
+DEBUG_TOOLBAR_PATCH_SETTINGS = False
```
Comment (OriHoch) [inline knesset/settings.py]: @daonb  - please remove / move before local_settings

File: less/bootstrap [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit da11bff2db35c1b82a9bb80df2c8628a77420ca6
+Subproject commit 51daa04f2b1a6e3eff0f880979d033a10165e3b3
```
Comment (OriHoch) [inline less/bootstrap]: @daonb - did you mean to update this?

**Comment by daonb** [comment]
I've fixed the PR

**Comment by daonb** [comment]
Seems like the marker is waiting for this. Please merge ASAP

File: static/css/app.css [hunk 3]
```diff
@@ -8089,3 +8089,6 @@ body.infinity-protocol #content-main > div > div.row > div.span3 {
   position: fixed;
   overflow: auto;
 }
+.tobecopied {
```
Comment (MeirKriheli) [inline static/css/app.css]: There are direct changes to app.css there.

It'll be overwritten when the less file are compiled. It should be done in less files

**Pull request merged**